cfme/configure/access_control/__init__.py

```diff
@@ -1435,14 +1435,14 @@ def quota(self):
             'num_vms': 'Allocated Number of Virtual Machines',
             'templates': 'Allocated Number of Templates'
         }
-
+        # This table does not work normally because of colspan, this is workaround for that problem.
```

Comment:
Table widget colspan implementation should support this, we need to take a look at what's happening here.  Please open an issue against wt.core so we can debug.